.credo.exs

```diff
@@ -31,7 +31,13 @@
           "apps/*/test/",
           "apps/*/web/"
         ],
-        excluded: [~r"/_build/", ~r"/deps/", ~r"/node_modules/"]
+        excluded: [
+          ~r"/_build/",
+          ~r"/deps/",
+          ~r"/node_modules/",
+          "lib/jellyfish_web.ex",
+          "lib/jellyfish_web/telemetry.ex"
```

Comment:
If the reason of ignoring `jellyfish_web.ex` and `telemetry.ex` is the requirement for specs for public functions, I would favor disabling ` Credo.Check.Readability.Specs` instead